proto/Cargo.toml

```diff
@@ -27,6 +27,7 @@ prost-types = "0.9"
 bytes = "1.1"
 tonic = "0.6"
 getrandom = { version = "0.2", features = ["js"] }
+serde = "1.0.130"
```

Comment:
```suggestion
serde = "1"
```

Reply:
> will avoid dependabot bothering us with updates.

It will anyway because the repo has `Cargo.lock` checked in.
Regardless, there is no need to depend on a semver patch version unless you want to make sure that e.g. a bugfix in the dependency is addressed since that specific version.

Reply:
Good point! I guess one of us copied the version from crates.io (which always shows the full semver), and we stuck to it for consistency. I just set it to `1.0` for all Cargo.toml's in the repo. https://github.com/informalsystems/ibc-rs/pull/1574/commits/a72ab391e34d00aab88fe307322f79edec0f69d2